src/shared/mkdir-label.c

```diff
@@ -4,6 +4,7 @@
 #include "selinux-util.h"
 #include "smack-util.h"
 #include "user-util.h"
+#include <sys/stat.h>
```

Comment:
order in a new block above the internal headers plz

Reply:
done